### ISSUE TITLE
Can not install on Centos 6.8

### ISSUE THREAD

**Issue by asdfgx64** [zfsonlinux/zfs#4932]
Hello,

we have issue with installation latest ZFS on our clean CentOS 6.8 installation. System is updated.

```
Error: Package: zfs-dkms-0.6.5.7-1.el6.noarch (zfs)
           Requires: dkms >= 2.2.0.3-20
           Available: dkms-2.2.0.3-2.zfs1.el6.noarch (zfs)
               dkms = 2.2.0.3-2.zfs1.el6
           Installing: dkms-2.2.0.3-14.zfs1.el6.noarch (zfs)
               dkms = 2.2.0.3-14.zfs1.el6
Error: Package: zfs-dkms-0.6.5.7-1.el6.noarch (zfs)
           Requires: dkms >= 2.2.0.3-20
           Available: dkms-2.2.0.3-2.zfs1.el6.noarch (zfs)
               dkms = 2.2.0.3-2.zfs1.el6
           Available: dkms-2.2.0.3-14.zfs1.el6.noarch (zfs)
               dkms = 2.2.0.3-14.zfs1.el6
```

**Comment by asdfgx64** [comment]
`# yum install dkms
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * base: ftp.energotel.sk
 * extras: ftp.energotel.sk
 * openvz-kernel-rhel6: openvz.website-solution.netkernel
 * openvz-utils: openvz.website-solution.netcurrent
 * rpmforge: mirror.vutbr.cz
 * updates: ftp.energotel.sk
rpmforge                                                                                                                                                                                                                                                | 1.9 kB     00:00
rpmforge/primary_db                                                                                                                                                                                                                                     | 2.7 MB     00:00
Resolving Dependencies
--> Running transaction check
---> Package dkms.noarch 0:2.2.0.3-14.zfs1.el6 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===============================================================================================================================================================================================================================================================================
 Package                                                       Arch                                                            Version                                                                      Repository                                                    Size
===============================================================================================================================================================================================================================================================================
Installing:
 dkms                                                          noarch                                                          2.2.0.3-14.zfs1.el6                                                          zfs                                                           74 k

Transaction Summary
===============================================================================================================================================================================================================================================================================
Install       1 Package(s)

Total download size: 74 k
Installed size: 209 k
Is this ok [y/N]: y
Downloading Packages:
http://archive.zfsonlinux.org/epel/6/kmod/x86_64/dkms-2.2.0.3-14.zfs1.el6.noarch.rpm: [Errno 14] PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"
Trying other mirror.
To address this issue please refer to the below knowledge base article

https://access.redhat.com/articles/1320623

If above article doesn't help to resolve this issue please open a ticket with Red Hat Support.



Error Downloading Packages:
  dkms-2.2.0.3-14.zfs1.el6.noarch: failure: dkms-2.2.0.3-14.zfs1.el6.noarch.rpm from zfs: [Errno 256] No more mirrors to try.
`

**Comment by behlendorf** [comment]
@asdfgx64 you need to install the EPEL repository for CentOS to get the latest version of dkms.  This is no longer provided by ZoL because a custom build is no longer needed.

```
sudo yum install epel-release
```
